src/ZoneServer/Skills/Handlers/Archer/TwinArrows.cs

```diff
@@ -62,9 +62,7 @@ private async void Attack(Skill skill, ICombatEntity caster, ICombatEntity desig
 
 			var skillHit = new SkillHitInfo(caster, designatedTarget, skill, skillHitResult, damageDelay, skillHitDelay);
 
-			Send.ZC_SKILL_READY(caster, skill, caster.Position, caster.Position);
```

Comment:
I think you might be misunderstanding what I'm trying to say. I didn't mean you should remove `ZC_SKILL_READY` and `ZC_SKILL_HIT_INFO`, but only replace `ZC_SKILL_MELEE_GROUND` with `ZC_SKILL_FORCE_TARGET`, because that's what the game usually sends. And you also still need to take a look at the aforementioned scriptable function for how multi-hit skills works, because they generally don't send two hits, but only one that has a special property set to animate a double-hit.